app/jinja_filters.py

```diff
@@ -79,11 +79,17 @@ def format_unit(
     # until removed from schema we substitute mass-tonne for mass-metric-ton before format unit
     measurement_unit = "mass-tonne" if unit == "mass-metric-ton" else unit
 
+    locale = (
+        DEFAULT_LOCALE
+        if str(flask_babel.get_locale()) in {"en", "eo"}
```

Comment:
How come this was needed for "eo" as well?

Reply:
This is a historical code, "eo" was a language code used for Ulsters Scots, as they didn't have an official one, that was supported during census, it uses same transforms as "en" but I guess it is not needed in here, so I will remove it.